src/main/java/org/springframework/data/elasticsearch/core/DefaultResultMapper.java

```diff
@@ -32,6 +32,7 @@
 import org.springframework.data.elasticsearch.annotations.ScriptedField;
 import org.springframework.data.elasticsearch.core.aggregation.AggregatedPage;
 import org.springframework.data.elasticsearch.core.aggregation.impl.AggregatedPageImpl;
+import org.springframework.data.elasticsearch.core.convert.MappingElasticsearchConverter;
```

Comment:
When cleaning up, we should merge `DefaultResultMapper` functionality into `MappingElasticsearchConverter`.

Reply:
yes, that's DATAES-630 to cleanup the other Mapper classes